.circleci/config.yml

```diff
@@ -145,13 +145,13 @@ commands:
       - run:
           name: Build for platform (docker build)
           command: |
-            make -C build/docker build DOCKER_SUFFIX=".<<parameters.platform>>" OSNICK=<<parameters.platform>> TEST=1 VERBOSE=1
+            make -C build/docker build DOCKER_SUFFIX=".<<parameters.platform>>" OSNICK=<<parameters.platform>> CIRCLE_BRANCH=${CIRCLE_BRANCH} CIRCLE_TAG=${CIRCLE_TAG} TEST=1 VERBOSE=1
```

Comment:
Don't we process `CIRCLE_BRANCH`/`CIRCLE_TAG` inside `mk/docker.rules`?
Also, do we still need an explicit `DOCKER_SUFFIX` here?

Reply:
Correct - removed in the latest push.